test/test_attention.py

```diff
@@ -106,6 +90,9 @@ def test_benchmark_masked(
     func = implementations[implementation]
     value = benchmark(func, **cast_args)
 
+    for _ in range(10):
```

Comment:
put this after for early fail ?

Reply:
and do we need that much run ?

Reply:
and can we have a different message for output stability ?

Reply:
* ok for early fail
* on Hazyresearch they put 10k, I think it matters to repeat stuff
* message for output stability would require to change the assert function, and right now we have enough info (we have the breaking line in pytest output), seems to be ok to me